lib/client.js

```diff
@@ -288,7 +289,7 @@ class Client {
     let hint = params.id_token_hint;
 
     if (hint instanceof TokenSet) {
-      assert(hint.id_token, 'id_token not present in TokenSet');
+      assert(hint.id_token, 'endSessionUrl() id_token not present in TokenSet');
```

Comment:
updated the error message here so it's source was transparent. I have done the same for other places where id_token is not set